test/config.test.js

```diff
@@ -270,6 +270,37 @@ describe('javascript config', () => {
 
 });
 
+
+describe('`logger` config', () => {
+    it('should exist', () => {
+        expect(config.logger).toBeDefined();
+    });
+
+    describe('`dir` it should exist', () => {
+        expect(config.logger.dir).toBeDefined();
```

Comment:
Doesn't the following test already cover the defined scenario? i.e. it'll fail if it doesn't match `'JavaScript'`.

Reply:
This one is different to the test below. The idea here is that it will fail should the object property `dir` not exist.  The second one is more of a value check, but I know what you mean, the value wouldn't pass if the `dir` isn't there in the first place so having it fail before that check makes it easier to spot what the actual fix, is what do you think?